.github/workflows/early_access.yml

```diff
@@ -15,8 +15,7 @@ jobs:
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
     - name: Set up JDK 11
-      uses: actions/setup-java@0ab4596768b603586c0de567f2430c30f5b0d2b0 # v3.13.0
-      with:
+      uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4.0.0
```

Comment:
Strange that this is version 4 and the other is 3.13. Oversight or intention? And why it isn't `@4`

Reply:
GH recommends pinning versions to the hash commit as (something like) v4 is effectively an alias that can silently be changed under the hood.